ecommerce/core/tests/test_models.py

```diff
@@ -163,14 +163,14 @@ def test_user_details(self):
         self.assertDictEqual(user.account_details(self.request), user_details)
 
     def test_user_details_uses_jwt(self):
-        """Verify user_details uses jwt from site configuration to call EdxRestApiClient."""
+        """Verify user_details uses jwt from site configuration when using the OAuthAPIClient."""
```

Comment:
Since the docstring is updated I would follow the style guide and rewrite a single docstring in a [multiple docstring format](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings)